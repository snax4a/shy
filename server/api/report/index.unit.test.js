/* globals describe, expect, test */
import router from './index';

describe('Report API Route Handlers:', () =>
  test('should have unchanged route handlers', () =>
    expect(router.stack).toMatchSnapshot()
  )
);
