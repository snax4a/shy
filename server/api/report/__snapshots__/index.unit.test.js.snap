// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report API Route Handlers: should have unchanged route handlers 1`] = `
Array [
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "get": true,
      },
      "path": "/",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "middleware",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
]
`;
